tests/functional/conftest.py

```diff
@@ -90,17 +90,17 @@ def _clean_database(engine):
 
 
 def _clean_elasticsearch(settings):
-    import elasticsearch
+    import elasticsearch1
 
-    conn = elasticsearch.Elasticsearch([settings['es.host']])
+    conn = elasticsearch1.Elasticsearch([settings['es.host']])
     conn.delete_by_query(index=settings['es.index'],
                          body={"query": {"match_all": {}}})
 
 
 def _drop_indices(settings):
-    import elasticsearch
+    import elasticsearch1
 
-    conn = elasticsearch.Elasticsearch([settings['es.host']])
+    conn = elasticsearch1.Elasticsearch([settings['es.host']])
 
     name = settings['es.index']
     if conn.indices.exists(index=name):
```

Comment:
Note I didn't mess with any of the tests. This is the place that I suspect @lyzadanger was talking about fixing and  @seanh is modifying in his PR (and in fact deleted) so I don't wanna mess with it. (I wasn't able to catch her to ask before she left :( ) LMK if you think differently!